CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - PR #545 Fix build to support using `clang` as the host compiler
 - PR #534 Fix `pool_memory_resource` failure when init and max pool sizes are equal
 - PR #546 Remove CUDA driver linking and correct NVTX macro.
+- PR #559 Fix align_down to only change unaligned values.
```

Comment:
```suggestion
- PR #559 Fix `align_down` to only change unaligned values.
```

Reply:
Thanks @codereport , but I don't think `bit_floor` will help here, since we aren't rounding to a power of 2, we are rounding to a multiple of a specific power of two (aligning).